plugins/services/src/js/service-configuration/ServiceLabelsConfigSection.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
```

Comment:
This is `LabelsServiceConfigSection.js` and `ServiceConfigLabelsSectionDisplay.js` combined